edm4hep.yaml

```diff
@@ -511,11 +511,13 @@ datatypes:
       - int32_t type // type of the raw data hit
       - int32_t quality // quality bit flag of the hit
       - float time [ns] // time of the hit
-      - float eDep [GeV] // energy deposited on the hit
-      - float eDepError [GeV] // error measured on eDep
-      - edm4hep::Vector3d position [mm] // point on the sensitive wire which is closest to the hit (center of the circle)
+      - float eDep [GeV] // energy deposited by the hit
+      - float eDepError [GeV] // error on eDep
       - edm4hep::Vector3d directionSW // direction of the sensitive wire (normal vector to the circle)
-      - float distanceToWire [mm] // distance hit-wire (radius of the circle)
+      - edm4hep::Vector3d position [mm] // point on the sensitive wire which is closest to the hit (center of the circle)
+      - double positionAlongWireError [mm] // error on the hit position along the wire direction
```

Comment:
How would the interpretation of the `positionAlongWireError` work in this case? IIUC I would need the `directionSW` to get the direction and then the `position` to get a point on the wire to get a reference point(?).

Reply:
Correct, the reference point on the wire is `position`, then to know "in which direction the error is defined" without accessing the geometry, you need `sensitiveWireDirection` (which I just renamed for more clarity)